src/bootstrap/builder.rs

```diff
@@ -1933,6 +1933,11 @@ impl<'a> Builder<'a> {
                 rustflags.arg("-Zvalidate-mir");
                 rustflags.arg(&format!("-Zmir-opt-level={}", mir_opt_level));
             }
+
+            if stage != 0 {
```

Comment:
```suggestion
            // cfg(not(bootstrap))
            if stage != 0 {
```
this helps t-release find it when they do the beta bump.